docs/tutorials/resource-migration.md

```diff
@@ -138,98 +110,92 @@ spec:
 ```
 
+
 >  **Note:**
 >
-> You should pay attention to two fields： 
+> You should pay attention to two fields：
 >
 > * `spec.conflictResolution: Overwrite`：the value must be [Overwrite](https://github.com/karmada-io/karmada/blob/master/docs/proposals/migration/design-of-seamless-cluster-migration-scheme.md#proposal).
 > * `spec.resourceSelectors`：selecting resources to migrate, you can define your custom [ResourceSelector](https://karmada.io/docs/userguide/scheduling/override-policy/#resource-selector).
 
-#### Step 3: Run the command
+#### Step 3: Run the Command
 
-Apply this PropagationPolicy to Karmada control plane. 
+Apply the above PropagationPolicy to the Karmada control plane:
 
 ```shell
-$ kubectl --context karmada-apiserver apply -f /tmp/pp-for-migrating-deployments-and-services.yaml
+$ kubectl --context karmada-apiserver apply -f /tmp/pp-for-migrating-deployments.yaml
 propagationpolicy.policy.karmada.io/migrate-pp created
 ```
 
 #### Step 4: Verification
 
 ```shell
-$ kubectl --context karmada-apiserver get deploy
-NAME           READY   UP-TO-DATE   AVAILABLE   AGE
-nginx-deploy   2/2     2            2           38s
-$ kubectl --context karmada-apiserver get rb
-NAME                      SCHEDULED   FULLYAPPLIED   AGE
-nginx-deploy-deployment   True        True           13s
-nginx-svc-service         True        True           13s
+$ karmadactl --karmada-context karmada-apiserver get deploy --operation-scope=all
+NAME           CLUSTER   READY   UP-TO-DATE   AVAILABLE   AGE   ADOPTION
+nginx-deploy   Karmada   4/2     4            4           34s   -
+nginx-deploy   member1   2/2     2            2           16m   Y
+nginx-deploy   member2   2/2     2            2           8s    Y
+
+$ karmadactl --karmada-context karmada-apiserver get pods --operation-scope=all
+NAME                            CLUSTER   READY   STATUS    RESTARTS   AGE
+nginx-deploy-54b9c68f67-4qjrz   member1   1/1     Running   0          16m
+nginx-deploy-54b9c68f67-f4qpm   member1   1/1     Running   0          16m
+nginx-deploy-54b9c68f67-f8kh7   member2   1/1     Running   0          10s
+nginx-deploy-54b9c68f67-vwhrk   member2   1/1     Running   0          10s
 ```
 
-You shall see the Deployments in Karmada are all ready and the `aggregatedStatus` of `ResourceBinding` is applied, 
-which means the existing resources in `member1` cluster has been taken over by Karmada. 
+You will see that all Deployments in Karmada are ready, among them:
 
-So far, you have finished the migration, isn't it so easy?
+* The `nginx-deploy` in the member1 cluster successfully taken over by Karmada:
+  * `AGE=16m` indicates the resource was taken over and not recreated, so the corresponding pods do not restart.
+  * `ADOPTION=Y` indicates that the resource is now managed by Karmada.
+* The member2 cluster, which previously had no `nginx-deploy`, has successfully received the deployment in replication mode, 
+  with `ADOPTION=Y` indicating it is now managed by Karmada.
```

Comment:
Deploying to members is not the intention of this documentation. 
This doc should focus on demonstrating migration.